PLEP-0007.rst

```diff
@@ -0,0 +1,189 @@
+| **dispersion**                                                               |
++------------------------------------------------------------------------------+
+| :directory: ``./plasmapy/dispersion/``                                       |
+| :access: ``plasmapy.dispersion``                                             |
+| :sub-package PLEP:                                                           |
+| :scope:                                                                      |
+|   | blah blah blah                                                           |
+|   | blah blah blah                                                           |
++------------------------------------------------------------------------------+
+| **examples**                                                                 |
++------------------------------------------------------------------------------+
+| :directory: ``./plasmapy/examples/``                                         |
+| :access: ``plasmapy.examples``                                               |
+| :sub-package PLEP:                                                           |
+| :scope:                                                                      |
+|   | blah blah blah                                                           |
+|   | blah blah blah                                                           |
++------------------------------------------------------------------------------+
+| **formulary**                                                                |
++------------------------------------------------------------------------------+
+| :directory: ``./plasmapy/formulary/``                                        |
+| :access: ``plasmapy.formulary``                                              |
+| :sub-package PLEP:                                                           |
+| :scope:                                                                      |
+|   A collection of functions, classes, and sub-packages that provide          |
+|   mathematical and scientific formulas for calculating physical parameters   |
+|   of various plasmas.                                                        |
++------------------------------------------------------------------------------+
+| **particle**                                                                 |
++------------------------------------------------------------------------------+
+| :directory: ``./plasmapy/particle/``                                         |
+| :access: ``plasmapy.particle``                                               |
+| :sub-package PLEP:                                                           |
+| :scope:                                                                      |
+|   | blah blah blah                                                           |
+|   | blah blah blah                                                           |
```

Comment:
Are we renaming `atomic`, then?

Reply:
Yes, the that's the current plan.  We all determined `atomic` was a little too specific whereas `particle` can be more encompassing.

Reply:
The name we use for this should encompass neutral atoms; ions; molecules; fundamental particles like electrons, positrons, muons, and neutrinos; charged dust grains; and custom particles.  

I'd prefer `plasmapy.particles` over `plasmapy.particle`, since it would reduce confusion to have a different pronunciation from the `Particle` class, and the subpackage would be describing the category of "particles".  

I've still been trying to think of alternatives to `plasmapy.particles` but I can't think of anything that would improve on it.

Reply:
`particles` sound like a good idea, or, er, multiple good ideas! :smile: